src/bitcoin/d/mod.rs

```diff
@@ -1262,7 +1262,7 @@ impl SyncProgress {
         let progress = roundup_progress(self.percentage);
         if progress == 1.0 && self.blocks != self.headers {
```

Comment:
not related to this PR:
```suggestion
        if progress == 1.0 && self.blocks < self.headers {
```
@jp1ac4 i don't think this is likely to happen as iiuc as of now it's only used w/ a bitcoind backend, but wdyt we should do if `self.blocks` > `self.headers`? we actually return 0.999 but maybe 1.0 should be more relevant?  (i'm not sure from a security POV, if returning 0.999 i guess we will stuck in the syncing process while if returning 1.0 we will assumed we are synced)

Reply:
Hmm I'm not sure really but I guess if they're not equal in either direction then something still needs to happen in terms of syncing.